### DISCUSSION TITLE
Do I upgrade to Frigate-HomeAssistant Integration 4.0.0

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [blakeblackshear/frigate#5945] [Ask A Question] [answered]
I understand from the notes for 0.12.0 RC2 that I need to upgrade my Frigate-HomeAssistant Integration from 3.0.1 to 4.0.0. However, Home Assistant will only upgrade it to 3.0.1. It looks like 4.0.0 is only available as 4.0.0-rc1. How do I get Home Assistant to install 4.0.0-rc1?

**Comment by NickM-27** [comment, marked as answer]
You use the 3 dots menu in HACS to enable beta builds. The integration should be fully released soon

**Reply by Cold-Lemonade** [reply]
@NickM-27 Thank you! I would never have figured that out. Many, many thanks.
